liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1051,8 +1128,22 @@ protected void doExecute() throws Exception {
             }
             runLibertyMojoDeploy();
         }
+        
+        // Check if we are using the exploded loose app functionality and save for checking later on. 
+        isExplodedLooseWarApp = LooseWarApplication.isExploded(project);
```

Comment:
From looking at code above, it seems we could be here for an `ear` (line 1015). Seems like you would need to account for that.

Reply:
Done